src/wiktextract/wiktwords.py

```diff
@@ -337,15 +339,30 @@ def main():
         args.inflections = True
         args.descendants = True
 
-    # Default to English and Translingual if language not specified.
-    if not args.language:
-        args.language = ["en", "mul"]
+    # Default to dump file language and Translingual if not specified.
+    capture_lang_codes = set()
```

Comment:
`WiktionaryConfig.capture_lang_codes` is changed to a `set`. Because we only check if a code is in the set and it also removes duplicated codes.